docs/markdown/flexible-plots.md

```diff
@@ -1,31 +1,53 @@
 ## Flexible plots
 
-By default, XYPlot requires a width and a height. There are times, however, when you'd like your chart to take all the space it can. For these cases, React-vis provides a helper function to make one or both dimensions of your chart flexible.
+By default, XYPlot requires a width and a height. There are times, however, when you'd like your chart to take all the space it can.
+For these cases, React-vis provides three different types of `XYPlot` with different flexible dimensions:
+
+  - FlexibleWidthXYPlot
+  - FlexibleHeightXYPlot
+  - FlexibleXYPlot
+
+and the associated helper functions that have been used to create these flexible components.
 
 <!-- INJECT:"FlexibleCharts" -->
 
 ```jsx
-const FlexibleWidth = makeWidthFlexible(XYPlot);
-const FlexibleHeight = makeHeightFlexible(XYPlot);
-const FlexibleVis = makeVisFlexible(XYPlot);
+import {
+  FlexibleXYPlot,
+  FlexibleWidthXYPlot,
+  FlexibleHeightXYPlot
+} from 'react-vis';
+
+// or
+import {
+  makeVisFlexible,
+  makeWidthFlexible,
+  makeHeightFlexible,
+} from 'react-vis';
+
+const FlexibleXYPlot = makeVisFlexible(XYPlot);
+const FlexibleWidthXYPlot = makeWidthFlexible(XYPlot);
+const FlexibleHeightXYPlot = makeHeightFlexible(XYPlot);
 ```
 
-makeWidthFlexible modifies XYPlot so that it no longer requires a width. It will take all the with in its container div. Likewise, makeHeightFlexible modifies XYPlot so that is no longer requires a height. Its height will be that of its container div. Finally, makeVisFlexible modifies XYPlot so that it no longer requires eiter a width and a height. Its dimensions will be that of its container.
+`makeWidthFlexible` modifies `XYPlot` so that it no longer requires a width.
```

Comment:
such a change would allow you to talk directly about just FlexibleWidthXYPlot directly here